applications/luci-app-upnp/root/usr/libexec/rpcd/luci.upnp

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env lua
```

Comment:
We could consider rewriting this RPC helper as shell script, but its no precondition for merging right now.